doc/developers/tips.rst

```diff
@@ -115,6 +172,11 @@ PR-WIP: Regression test needed
 
         Please add a [non-regression test](https://en.wikipedia.org/wiki/Non-regression_testing) that would fail at master but pass in this PR.
 
+PR-WIP: PEP8
+    ::
+
+        You have some [PEP8](https://www.python.org/dev/peps/pep-0008/) violations, whose details you can see in Travis CI. It might be worth configuring your code editor to check for such errors on the fly, so you can catch them before committing.
```

Comment:
I know on the fly check in the editor is the best practice, but might be little bit difficult for new comers? Will it be better to provide something simpler, e.g., `pip install flake8; flake8 XXX.py`?

Reply:
For new comers, I think the Travis flake8 error is good enough.